sample/src/main/java/com/auth0/sample/DatabaseLoginFragment.kt

```diff
@@ -40,6 +51,10 @@ class DatabaseLoginFragment : Fragment() {
         savedInstanceState: Bundle?
     ): View {
         val binding = FragmentDatabaseLoginBinding.inflate(inflater, container, false)
+        val authentication = AuthenticationAPIClient(account)
+        val storage = SharedPreferencesStorage(requireContext())
+        manager = SecureCredentialsManager(requireContext(), authentication, storage)
+        manager.requireAuthentication(requireActivity(), 201, "Hi Hello", "Poovam");
```

Comment:
👋 

You've left some testing code in here

Reply:
My bad 🤦  Great catch though. I have fixed it now, there should be zero change in that file